pkg/kv/kvclient/kvcoord/dist_sender_test.go

```diff
@@ -1370,8 +1370,8 @@ func TestEvictCacheOnError(t *testing.T) {
 	}
 
 	rangeMismachErr := roachpb.NewRangeKeyMismatchError(
-		context.Background(), nil, nil, &lhs, nil /* lease */)
-	rangeMismachErr.AppendRangeInfo(context.Background(), rhs, roachpb.Lease{})
+		context.Background(), nil, nil, &lhs, nil /* lease */, roachpb.LAG_BY_CLUSTER_SETTING)
```

Comment:
[nit] Should we change the constructor here to take in a pointer instead? That way the caller doesn't need to bother with the verbose default. Alternatively, have another constructor for NewRangeKeyMismatchError that takes in an explicit closed-ts policy.

Reply:
Updated to a new constructor, one including the closed timestamp policy and another not.